next.clayui.com/content/docs/foundations/composing.md

```diff
@@ -3,4 +3,146 @@ title: "Composing"
 description: "Component composition philosophy"
 ---
 
-This is a stub. Help our community expand it.
\ No newline at end of file
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+- [Composition](#composition)
+- [Philosophy](#philosophy)
+    - [Low-level components](#low-level-components)
+	- [High-level components](#high-level-components)
```

Comment:
Indentation is off here. I think if your editor is configured to used spaces over tabs, it gets wonky when prettier formats.